.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+env:
+  GH_TOKEN: ${{ github.token }}      
+
+jobs:
+    release:
+        name: Create new release
+        runs-on: ${{ matrix.os }}
+        strategy:
+          matrix:
+            include:
+              - os: ubuntu-latest
+                filename_suffix: "${{ github.ref_name }}-linux-amd64"
+        steps:
+          - uses: actions/checkout@v4
+          - uses: dtolnay/rust-toolchain@stable
+          - run: cargo build --release --locked ${{ matrix.flags }}
+          - run: gh release create ${{ github.ref }} --draft --title "Uhyve ${{ github.ref_name }}"
```

Comment:
This should be done before building because we want to avoid the race if we add more targets.